1-exercises/D-for-in-loops/exercise-1.js

```diff
@@ -11,8 +11,10 @@ let salaries = {
   monica: 55000,
   rachel: 90000,
 };
-
 // WRITE BELOW THIS LINE
+for (const item in salaries){
+  if (salaries[item] > 5000) console.log(`${item}: ${salaries[item]}`);
```

Comment:
Missing a zero on the amount but you've got the concept that's what matters. would prefer if you always used if with curly brackets i.e.
```
if () {

}
```
but that's my preference